pkg/app/piped/cloudprovider/lambda/client.go

```diff
@@ -108,56 +92,34 @@ func (c *client) IsFunctionExist(ctx context.Context, name string) (bool, error)
 	input := &lambda.GetFunctionInput{
 		FunctionName: aws.String(name),
 	}
-	_, err := c.client.GetFunctionWithContext(ctx, input)
+	_, err := c.client.GetFunction(ctx, input)
 	if err != nil {
-		if aerr, ok := err.(awserr.Error); ok {
-			switch aerr.Code() {
-			case lambda.ErrCodeInvalidParameterValueException:
-				return false, fmt.Errorf("invalid parameter given: %w", err)
-			case lambda.ErrCodeServiceException:
-				return false, fmt.Errorf("aws lambda service encountered an internal error: %w", err)
-			case lambda.ErrCodeTooManyRequestsException:
-				return false, fmt.Errorf("request throughput limit was exceeded: %w", err)
+		var nfe *types.ResourceNotFoundException
+		if errors.As(err, &nfe) {
 			// Only in case ResourceNotFound error occurred, the FunctionName is available for create so do not raise error.
-			case lambda.ErrCodeResourceNotFoundException:
-				return false, nil
-			}
+			return false, nil
 		}
-		return false, fmt.Errorf("unknown error given: %w", err)
+		return false, err
 	}
 	return true, nil
 }
 
 func (c *client) CreateFunction(ctx context.Context, fm FunctionManifest) error {
 	input := &lambda.CreateFunctionInput{
-		Code: &lambda.FunctionCode{
+		Code: &types.FunctionCode{
 			ImageUri: aws.String(fm.Spec.ImageURI),
 		},
-		PackageType:  aws.String("Image"),
+		PackageType:  types.PackageType("Image"),
 		Role:         aws.String(fm.Spec.Role),
```

Comment:
But here they are still using the shared one. ww

Reply:
🤣 Yep, and mixing with some `HogeInput` and `FugaOutput` types for API functions.